vllm/v1/worker/tpu_model_runner.py

```diff
@@ -263,10 +263,8 @@ def _update_states(self, scheduler_output: "SchedulerOutput") -> bool:
             req_id = new_req_data.req_id
             sampling_params = new_req_data.sampling_params
             if sampling_params.sampling_type == SamplingType.RANDOM_SEED:
-                generator = torch.Generator(device=self.device)
-                generator.manual_seed(sampling_params.seed)
-            else:
-                generator = None
+                logger.warning("Torch XLA does not support per-request seed."
+                               "Seed {sampling_params.seed} will be ignored")
```

Comment:
I think we might want to raise an error so the request will fail rather than ignore user specified seed

Reply:
@NickLucche why is it marked as resolved? Seems it's still a warning.